### PR TITLE
Jhbate fix errors regression

### PR DESCRIPTION
**Pull request by jh-bate**
@darinkrauss this uses the updated code from https://github.com/tidepool-org/amoeba/pull/6 and ensures the errors are getting sent back on a usable format

### CODE REVIEW

**Comment by darinkrauss** [comment]
@jh-bate Is this all to make sure Jellyfish returns the expect shape of the error and the front-end expects?  If so, then @jebeck probably needs to sign-off on this one.

File: lib/dataBroker.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ module.exports = function(streamDAO) {
             if (!schema.isValidVersion(versionNum, uploadConfig.minimumUploaderVersion)) {
               return cb({
                 statusCode: 400,
-                text: 'The minimum supported version is ['+uploadConfig.minimumUploaderVersion+']. Version ['+datum.version+'] is no longer supported.',
+                message: 'The minimum supported version is ['+uploadConfig.minimumUploaderVersion+']. Version ['+datum.version+'] is no longer supported.',
```
Comment (jebeck) [inline lib/dataBroker.js]: Perhaps this change is what's causing the uploader to fail to redirect to the "you need to update your uploader" when the version mismatch happens? Dunno, but the advantage of keeping it the same could be that versions of the uploader that are _already_ out in the wild will also benefit from the fix(es).

Maybe it's worth testing the prod uploader against new jellyfish under version-mismatch conditions?

Comment (jh-bate) [inline lib/dataBroker.js]: @jebeck and I discussed and decision was fixing the original issue of errors not being surface was the focus so I will leave this as updated.

**Comment by jebeck** [comment]
One last comment, which is just that we should try to get this out first before publishing the uploader with coordinated fixes, right? Let's make sure we don't introduce any (further) regressions in the current prod uploader with jellyfish. I *think* that's pretty much impossible, but let's remember to test. CC @darinkrauss

**Comment by jh-bate** [comment]
@jebeck @darinkrauss yeah will do some coordinating with @gniezen and @NSRiggall about the testing and release of this change

**Pull request merged**